cockroachdb/values.yaml

```diff
@@ -134,6 +134,16 @@ conf:
   # CockroachDB's port to listen to HTTP requests.
   http-port: 8080
 
+  # CockroachDB's storage configuration https://www.cockroachlabs.com/docs/v20.2/cockroach-start.html#storage
```

Comment:
Maybe mention that this is for supplying values to the optional `--store` field, and mention that this is only needed if `storage.persistentVolume.enabled` == `false`?

Reply:
No, you can use it with disabled PVs. Because of inmem mode

Reply:
Why? You can use for both persistent and inmem

Reply:
I see a reason to use it for in-memory or if using `hostPath`, but I don't really know why you would use it when using persistent volumes. But you can still use it for persistent volumes if you want.

Reply:
@DuskEagle Hmm, my main usecase is for persistent deployments to have a storage limit.

Example:
I have 3 nodes, 100Gb disks
One cluster with 10Gb PVs
CRDB will think he can use all 100Gb because of 100% default storage limit, but it is not true